source/_integrations/magichome.markdown

```diff
@@ -0,0 +1,57 @@
+---
+title: "MagicHome"
+description: "Instructions on how to setup the MagicHome hub within Home Assistant."
+logo: magichome.png
+ha_category:
+  - Light
+  - Switch
+ha_iot_class: Cloud Polling
+ha_release: 0.103
```

Comment:
`ha_domain` is missing.

Reply:
Is there any documentation describing ha_domain?

Reply:
`ha_domain` must have the same value as the `domain` in the manifest.json file of the integration. A PR to add details for `ha_domain` to our developer documentation, is pending.

Reply:
Thanks!